packages/trader/src/AppV2/Components/TradeParameters/RiskManagement/take-profit-and-stop-loss-input.tsx

```diff
@@ -92,6 +92,26 @@ const TakeProfitAndStopLossInput = ({
     // Storing data from validation params (proposal) in state in case if we got a validation error from API and proposal stop streaming
     const [info, setInfo] = React.useState<Record<string, string | undefined>>({ min_value, max_value });
 
+    const new_req = getProposalRequestObject({
+        is_take_profit_input,
+        is_enabled,
+        should_set_validation_params,
+        trade_store,
+        trade_type: Object.keys(trade_types)[0],
+        new_input_value,
+    });
+
+    const { data: response } = useDtraderQuery<Parameters<TOnProposalResponse>[0]>(
+        ['proposal', new_input_value ?? ''],
+        {
+            ...new_req,
+            subscribe: undefined,
```

Comment:
`subscribe:0` is trowing error

Reply:
you can do this also
delete new_req.subscribe from the object

Reply:
Ohh, ye, I forgot about this opportunity🙈
Thank you, will apply